packages/nc-gui-v2/components/webhook/List.vue

```diff
@@ -0,0 +1,98 @@
+<script setup lang="ts">
+import { useToast } from 'vue-toastification'
+import { onMounted } from '@vue/runtime-core'
+import { MetaInj } from '~/context'
+import MdiHookIcon from '~icons/mdi/hook'
+import MdiDeleteOutlineIcon from '~icons/mdi/delete-outline'
+
+const emit = defineEmits(['edit'])
+
+const { $api, $e } = useNuxtApp()
+
+const toast = useToast()
+
+const hooks = ref([])
+const meta = inject(MetaInj)
+
+async function loadHooksList() {
+  try {
+    const hookList = (await $api.dbTableWebhook.list(meta?.value.id as string)).list as any
+    hooks.value = hookList.map((hook: Record<string, any>) => {
```

Comment:
If you cast `.list` to `Record<string, any>[]` you can skip casting `hook: Record<string, any>`